lib/chat_api/users.ex

```diff
@@ -113,4 +100,83 @@ defmodule ChatApi.UserProfiles do
   def change_user_profile(%UserProfile{} = user_profile, attrs \\ %{}) do
     UserProfile.changeset(user_profile, attrs)
   end
+
+  def get_user_settings(user_id) do
+    UserSettings |> where(user_id: ^user_id) |> Repo.one()
+  end
+
+  def create_or_update_settings(user_id, params) do
```

Comment:
good idea, i think it's not too urgent though so i'll handle that in a separate ticket because we did something similar with our `user_profiles` and i'd like to update both :)

Reply:
https://github.com/papercups-io/papercups/issues/85